--- old	2023-03-23 14:17:42.536656160 +0100
+++ new	2023-03-24 23:07:08.186018906 +0100
@@ -140,6 +140,11 @@

   warn    set acl_m3    = no
  
+  warn    !authenticated = *
+          hosts          = !+relay_from_hosts
+          condition     = ${lookup{$local_part@$domain}lsearch{/etc/exim4/domains/$domain/aliases}{true}{false}}
+          set acl_m3    = yes
+
   deny    message       = Restricted characters in address
           domains       = +local_domains
           local_parts   = ^[.] : ^.*[@%!/|]
